setuptools/command/easy_install.py

```diff
@@ -2187,7 +2187,7 @@ def get_args(cls, dist, header=None):
         spec = str(dist.as_requirement())
         for type_ in 'console', 'gui':
             group = type_ + '_scripts'
-            for name, ep in dist.get_entry_map(group).items():
+            for name in dist.get_entry_map(group).keys():
```

Comment:
If we are not using the values, then we can just iterate over the original object no? There would be no reason to call `.keys()`, right?

Reply:
yeah iterating over a dict should be the same as iterating over its keys since we don't do anything special with the view-object